documentation/specs/src/interoperability/ethereum-bridge/proofs.md

```diff
@@ -0,0 +1,88 @@
+# Proofs
+
+A proof for the bridge is a quorum of signatures by a valid validator set
+attached to a message understandable to the Ethereum smart contracts. For
```

Comment:
question: do proofs only include >2/3 of the voting power worth of signatures, or do they require a signature from every validator in the set?